src/ontology/mondo-edit.obo

```diff
@@ -571264,7 +571264,7 @@ xref: MEDGEN:1875015 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: OMIM:620990 {source="MONDO:equivalentTo"}
 xref: UMLS:C5975485 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1875015"}
 is_a: MONDO:0009049 {source="OMIM:620990", source="https://orcid.org/0000-0002-4142-7153"} ! Cushing syndrome due to macronodular adrenal hyperplasia
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29079 {source="OMIM:620990"} ! KDM1A
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29079 ! KDM1A
```

Comment:
removed because omim tags this as a mutation that contributes to susceptibility to multifactorial disorders https://omim.org/entry/620990 and https://omim.org/help/faq#1_6

Reply:
same as above